wcivf/apps/people/migrations/0028_person_pk_to_int.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.15 on 2018-08-29 10:22
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def drop_varchar_pattern_ops_index(apps, schemaEditor):
+    models = [
+        apps.get_model("mentions", "mention_people"),
+        apps.get_model("mentions", "mention"),
+        apps.get_model("people", "AssociatedCompany"),
+        apps.get_model("people", "Person"),
+        apps.get_model("peoplecvs", "CV"),
+        apps.get_model("pledges", "CandidatePledge"),
+        apps.get_model("leaflets", "Leaflet"),
+    ]
+    indexed_to_delete = []
```

Comment:
Should this var name be `indexes_to_delete` rather than `indexed_to_delete`

Reply:
Spot 👀  Fixed and squashed